app/styles/app.scss

```diff
@@ -49,3 +49,4 @@
 @import 'mixins';
 @import 'newcomponents';
 @import 'layout/layout';
+@import 'components/charts';
```

Comment:
Please move this into newcomponents.  Everything is slowly moving there and no new stuff should go into `components`